packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -263,10 +266,10 @@ export const AddressStruct: Describe<AddressElement> = element('Address', {
  * A struct for the {@link BoxElement} type.
  */
 export const BoxStruct: Describe<BoxElement> = element('Box', {
-  children: maybeArray(
+  children: children(
     // eslint-disable-next-line @typescript-eslint/no-use-before-define
-    nullable(lazy(() => BoxChildStruct)),
-  ) as unknown as Struct<MaybeArray<GenericSnapElement | null>, null>,
+    [lazy(() => BoxChildStruct), boolean()],
```

Comment:
Can't we omit `boolean()` here?

Reply:
Good catch, forgot to update this after simplifying the types/validation with `children`.